setup.py

```diff
@@ -27,7 +27,7 @@
     "numpy",
     "flake8",
     "yanc",
-    "remotecv",
+    # "remotecv==2.2.3",
```

Comment:
This should be uncommented when release is published. See https://github.com/thumbor/thumbor/pull/1209/files#r300174816